dist/pica.js

```diff
@@ -1857,8 +1857,8 @@ Pica.prototype.resize = function (from, to, options) {
     opts = assign(opts, options);
   }
 
-  opts.toWidth = to.width;
-  opts.toHeight = to.height;
+  opts.toWidth = to.width || 1;
+  opts.toHeight = to.height || 1;
```

Comment:
I don't thinkthat's really needed. Also, it's very bad idea to magically adjust input params.

Reply:
So how about warning or throwing error when we found invalid args,e.g., when output  size <= 0?